xpsi/module_generator.py

```diff
@@ -1433,14 +1446,20 @@ def __call__(self, parser, namespace, values, option_string=None):
 if args.background_shared_class:
     module += (
     '''
-from {0} import CustomBackground
+if __name__ == '__main__':
+    from {0} import CustomBackground
+else:
+    from .{0} import CustomBackground
     '''.format(args.custom_background_module)
     )
 elif args.background_shared_class:
     for _instrument in args.instruments:
         module += (
         '''
-from {0} import {1}_CustomBackground
+if __name__ == '__main__':
+    from .{0} import {1}_CustomBackground
```

Comment:
Probably no dot should be here, right?

Reply:
And actually, it probably should be:
```
if __name__ == '__main__':
  from {0} import {1}_CustomBackground
          '''.format(args.custom_background_module,
                     _instrument)
else:
  from .{0} import {1}_CustomBackground
          '''.format(args.custom_background_module,
                     _instrument)
```

Reply:
Ah yes that's correct. The `else` condition is fine in the commit. The `if` condition has the typo that you caught

Reply:
I pushed a commit to fix that.